.github/workflows/main.yml

```
@@ -1 +1,52 @@
+name: CI
 
+on: [pull_request]
+
+jobs:
+  phpstan:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./app
+    steps:
+      - uses: actions/checkout@v4
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+      - run: composer install --no-scripts --no-cache
+
+      - name: PHPStan Static Analysis
+        run: ./vendor/bin/phpstan analyse src/
```

Pas besoin de préciser `src/` puisqu’il est déjà présent dans le fichier `phpstan.neon` (comme ça on aura qu’un seul endroit à changer si on rajoute des dossiers à analyser).

Il faudrait aussi ajouter l’option `-v`, ça permet de connaître le nom de la règle qui échoue (pratique si on veut l’ignorer)

```suggestion
        run: ./vendor/bin/phpstan analyse -v
```